linode/manager
# M3-5117: Remove __data__/types.ts and update affected files

## Description
Removes `__data__/types.ts` and updates the files (mostly tests) that used it accordingly by bringing back `cachedData/types.json` (see https://github.com/linode/manager/pull/7777/files) to use instead.

## How to test
Ensure that `yarn up` works with no TS errors; run the affected tests and ensure everything passes –

```
yarn test typesHelpers.test.ts
```

```
yarn test GetSearchEntities.test.ts
```

```
yarn test ResizeNodePoolDrawer.test.ts
```

```
yarn test BackupsTable.test.ts
```

```
yarn test BackupsDrawer.test.ts
```

```
yarn test BackupLinodes.test.ts
```
